File: 基礎題庫/Contents/c315/c315.c
#include<stdio.h>

int main(void)
{
	int n, ctrl, step;
	
	while(scanf("%d", &n) != EOF)
	{
		int x = 0;
		int y = 0;
		
		for(int i = 0; i < n; i++)
		{
			scanf("%d %d", &ctrl, &step);
			
			switch(ctrl)
			{
				case 0:
					y += step;
					break;
				case 1:
					x += step;
					break;
				case 2:
					y -= step;
					break;
				case 3:
					x -= step;
					break;
			}
		}
		
		printf("%d %d\n", x, y);
	}
	
	return 0;
}

File: 基礎題庫/Contents/c315/c315.py
import sys

for n in sys.stdin:
    n = int(n)

    x = y = 0
    for i in range(n):
        ctrl, step = map(int, input().split())

        if ctrl == 0:
            y += step
        elif ctrl == 1:
            x += step
        elif ctrl == 2:
            y -= step
        elif ctrl == 3:
            x -= step
    
    print(x, y)
